### ISSUE TITLE
Anti air as defense to missiles

### ISSUE THREAD

**Issue by HaphLife** [Anuken/Mindustry-Suggestions#3999]
### Describe what you would like changed, and why.

This is specifically regarding the missile shown in #dev-previews. [(shown here)](https://discord.com/channels/391020510269669376/513178557653188609/991822719509336225) Given that it is described as “not balanced” I presume that discussion about how to balance it might be reasonable.

### Describe the changes you want to propose. Include possible alternatives.

One possible way to balance it would be to allow anti-air to shoot down missiles. It might be interesting to make the missiles fast and low health (compared to units), so that rather than preferring hard hitting AA, turrets with higher turn speed are meta. This would make more AA options “useful” in the endgame.

Another related option, the different ammo types for the missile turret could allow the user to choose between hard hitting and slower (perhaps surge alloy), and weaker and faster (perhaps phase), so that you can choose a faster missile in cases where you need to outrun the enemy AA and slow when you can punish gaps in their defenses. (If you want more option suggestions, a material that gets many missiles per item but they are weak and slow, and incendiary missiles would be cool) Fast missiles and large explosions are both cool, but if they are made mutually exclusive, the turret is more balanced.

**TL;DR:**
Idea 1: fast rotating AA counters missiles.
Idea 2: missiles are only good at one thing, depending on their ammo type.

**Comment by LampbuIb** [comment]
Anti air capable turrets are already able to shoot down missiles from erekir.

**Comment by HaphLife** [comment]
> Anti air capable turrets are already able to shoot down missiles from erekir.

Cool, I expected that is something that’d get added, but I just wanted to point out that it would be possible to intentionally design interaction between aa gun rotation speed/bullet speed, and missile speed, if tuned appropriately.
